add smeared, projected PU for 2017 data

capalmer85 opened the pull request:
@klute, @franzoni, @davidlange6 requested PR.

capalmer85 committed:
add smeared, projected PU for 2017 data

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/PR-20965/1532

cmsbuild commented:
A new Pull Request was created by  @capalmer85  for master.

It involves the following packages:

Configuration/StandardSequences
SimGeneral/MixingModule


@cmsbuild, @civanch, @franzoni, @mdhildreth, @davidlange6 can you please review it and eventually sign? Thanks.
@ghellwig, @makortel, @felicepantaleo, @GiacomoSguazzoni, @rovere, @VinInn, @Martin-Grunewald, @ebrondol, @dgulhan this is something you requested to watch as well.
@davidlange6, @slava77 you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

davidlange6 commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 654bb63f05a23aca5814bb0b84e44c008398ce92
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20965/23870/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/af68bef39b8e9de9203b1ff8c74ff6fd3a432a02
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20965/23870/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20965/23870/git-merge-result

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20965/23870/summary.html

Comparison Summary:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 26
 - DQMHistoTests: Total histograms compared: 2795245
 - DQMHistoTests: Total failures: 120
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 2794954
 - DQMHistoTests: Total skipped: 171
 - DQMHistoTests: Total Missing objects: 0
 - Checked 107 log files, 10 edm output root files, 26 DQM output files

civanch commented:
+1

davidlange6 committed:
Update mix_2017_25ns_WinterMC_PUScenarioV1_PoissonOOTPU_cfi.py

use probFunction_25ns_PoissonOOTPU_cfi as basis instead of repeating lots of config

davidlange6 commented:
merge

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/PR-20965/1588

cmsbuild commented:
Pull request #20965 was updated. @cmsbuild, @civanch, @franzoni, @mdhildreth, @davidlange6 can you please check and sign again.
